translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionIterableAssertion.kt

```diff
@@ -14,6 +14,7 @@ enum class DescriptionIterableAssertion(override val value: String) : StringBase
     AT_MOST("is at most"),
     CONTAINS("contains"),
     CONTAINS_NOT("does not contain"),
+    EMPTY_ITERABLE("!! cannot be determined, empty Iterable"),
```

Comment:
There should be NO_ELEMENTS by now which provides the de_CH translation in addition. Reuse it